### PR TITLE
Added Account Duration/Experience Trophy

### PR DESCRIPTION
**Pull request by bhavberi**
_Addition_: Trophy named Experience
_Conditioning_: Based on the Number of Days since account creation
_Points Calculation_: Number of Days/100 & rounded down
_Rank Updates_: Kind of ~~Linear~~ exponential

The output of `deno lint --unstable`:
Screenshot from 2023-05-16 13-11-11

For username=`ryo-ma`
image

_FYI_: Based on a comment too - https://github.com/ryo-ma/github-profile-trophy/pull/174#pullrequestreview-1248735956

Hope I can get this PR merged. (Please let me know if you want any changes in the rank titles or conditionals for the ranks)
~ @bhavberi

### CODE REVIEW

**Review by MitkoVtori — APPROVED** [review]
I would like that trophy!

**Comment by CSFelix** [comment]
Great idea!!

**Comment by AlexcastroDev** [comment]
> _Addition_: Trophy named Experience _Conditioning_: Based on the Number of Days since account creation _Points Calculation_: Number of Days/100 & rounded down _Rank Updates_: Kind of ~Linear~ exponential
> 
> The output of `deno lint --unstable`: Screenshot from 2023-05-16 13-11-11
> 
> For username=`ryo-ma` image
> 
> _FYI_: Based on a comment too - [#174 (review)](https://github.com/ryo-ma/github-profile-trophy/pull/174#pullrequestreview-1248735956)
> 
> Hope I can get this PR merged. (Please let me know if you want any changes in the rank titles or conditionals for the ranks) ~ @bhavberi

Congrats !! 

Sorry for wait long time for review. I will try help @ryo-ma.

Ryota, can you review this  ?

**Review by ivan-developer-01 — APPROVED** [review]

**Comment by ryo-ma** [comment]
@bhavberi  can you rebase?

**Comment by bhavberi** [comment]
@ryo-ma Can you please check it once?

**Comment by ryo-ma** [comment]
@bhavberi I'll review it after work today.

File: src/Services/GithubApiService.ts [hunk 1]
```diff
@@ -146,7 +146,10 @@ export class GithubApiService extends GithubRepository {
         Logger.error(error);
       }
 
-      return new ServiceError("Rate limit exceeded", EServiceKindError.RATE_LIMIT);
+      return new ServiceError(
```
Comment (ryo-ma) [inline src/Services/GithubApiService.ts:149]: This is an unnecessary correction for this PR.

Comment (ryo-ma) [reply src/Services/GithubApiService.ts:149]: Please remove from commit.

Comment (AlexcastroDev) [reply src/Services/GithubApiService.ts:149]: I think the deno format does that

Comment (bhavberi) [reply src/Services/GithubApiService.ts]: Restored this specific part as original.

**Comment by AlexcastroDev** [comment]
@bhavberi I am sorry, this format was my fault, can  you rebase with master ?
I updated the master code with formated fixed.

I am sorry again.

**Comment by bhavberi** [comment]
Done :+1:

**Comment by ryo-ma** [comment]
Sorry, please one more rebase. I will merge this PR next.

**Comment by ryo-ma** [comment]
Thank you for your PR. I have merged.

**Review by SmashedFrenzy16 — COMMENTED** [review]
Great idea for a trophy!

**Pull request merged**